tests/integration/acp/index/query_with_relation_test.go

```diff
@@ -111,12 +111,14 @@ func TestACPWithIndex_UponQueryingPrivateOneToManyRelatedDocWithoutIdentity_Shou
 							}
 						}
 					}`,
-				Results: []map[string]any{
-					{
-						"name": "John Grisham",
-						"published": []map[string]any{
-							{
-								"name": "Painted House",
+				Results: map[string]any{
+					"Author": []map[string]any{
```

Comment:
praise: you did it all by hand? Wow. That must have been tedious.

Reply:
I'm not gonna lie, It was a lot of work, but it had to be done.